hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/test/resources/capacity-scheduler.xml

```diff
@@ -97,6 +97,15 @@
     </description>
   </property>
 
+  <property>
+    <name>yarn.scheduler.capacity.root.default.reservable</name>
```

Comment:
Is this a merge issue?

Reply:
Hi, @goiri , testListReservations junit test needs the queue to be reservable, I set root.default reservable = true.

Reply:
I newly set the `root.decided` queue for Junit Test.